tests/test_hotkeys.py

```
@@ -0,0 +1,620 @@
+        yield mock_kb
+
+@pytest.fixture
+def mock_logger():
+    """Fixture providing a mocked logger."""
+    with patch('agor.tools.hotkeys.logger') as mock_log:
+        yield mock_log
+
+@pytest.fixture(autouse=True)
+def reset_global_manager():
+    """Automatically reset global manager state before each test."""
+    global_manager.stop()
```

Ruff (0.11.9)

Undefined name `global_manager`

(F821)